### DISCUSSION TITLE
Fix bridge.yml

### DISCUSSION THREAD

**Discussion by deep-soft** [rustdesk/rustdesk#4099] [General]
Just received error on last build, action bridge.yml:
# -----------------
Install prerequisites

Run sudo apt update -y

WARNING: apt does not have a stable CLI interface. Use with caution in scripts.

Hit:1 http://azure.archive.ubuntu.com/ubuntu focal InRelease
Get:2 http://azure.archive.ubuntu.com/ubuntu focal-updates InRelease [114 kB]
Get:3 http://azure.archive.ubuntu.com/ubuntu focal-backports InRelease [108 kB]
Get:4 http://azure.archive.ubuntu.com/ubuntu focal-security InRelease [114 kB]
Ign:5 https://packages.microsoft.com/ubuntu/20.04/prod focal InRelease
Err:6 https://packages.microsoft.com/ubuntu/20.04/prod focal Release
  Certificate verification failed: The certificate is NOT trusted. The certificate chain uses expired certificate.  Could not handshake: Error in the certificate verification. [IP: 13.90.21.104 443]
Hit:7 http://ppa.launchpad.net/ubuntu-toolchain-r/test/ubuntu focal InRelease
Get:8 http://azure.archive.ubuntu.com/ubuntu focal-updates/main amd64 Packages [2497 kB]
Get:9 http://azure.archive.ubuntu.com/ubuntu focal-updates/main Translation-en [424 kB]
Get:10 http://azure.archive.ubuntu.com/ubuntu focal-updates/main amd64 c-n-f Metadata [16.4 kB]
Get:11 http://azure.archive.ubuntu.com/ubuntu focal-updates/restricted amd64 Packages [1776 kB]
Get:12 http://azure.archive.ubuntu.com/ubuntu focal-updates/restricted Translation-en [249 kB]
Get:13 http://azure.archive.ubuntu.com/ubuntu focal-updates/restricted amd64 c-n-f Metadata [636 B]
Get:14 http://azure.archive.ubuntu.com/ubuntu focal-updates/universe amd64 Packages [1051 kB]
Get:15 http://azure.archive.ubuntu.com/ubuntu focal-updates/universe Translation-en [248 kB]
Get:16 http://azure.archive.ubuntu.com/ubuntu focal-updates/universe amd64 c-n-f Metadata [24.2 kB]
Get:17 http://azure.archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 Packages [25.2 kB]
Get:18 http://azure.archive.ubuntu.com/ubuntu focal-updates/multiverse amd64 c-n-f Metadata [612 B]
Get:19 http://azure.archive.ubuntu.com/ubuntu focal-security/main amd64 Packages [2085 kB]
Get:20 http://azure.archive.ubuntu.com/ubuntu focal-security/main amd64 c-n-f Metadata [12.5 kB]
Get:21 http://azure.archive.ubuntu.com/ubuntu focal-security/universe amd64 Packages [821 kB]
Get:22 http://azure.archive.ubuntu.com/ubuntu focal-security/universe amd64 c-n-f Metadata [17.6 kB]
Reading package lists...
E: The repository 'https://packages.microsoft.com/ubuntu/20.04/prod focal Release' no longer has a Release file.
Error: Process completed with exit code 100.

# -----------------

Fix bridge.yml:
insert
`          sudo apt install ca-certificates -y`
before
`          sudo apt update -y
`
see below:
```
      - name: Install prerequisites
        run: |
          sudo apt install ca-certificates -y
          sudo apt update -y
          sudo apt install -y g++ gcc git curl wget nasm yasm libgtk-3-dev clang cmake libclang-dev ninja-build llvm-dev libclang-10-dev llvm-10-dev pkg-config
```

**Comment by rustdesk** [comment]
add it then please. https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request

**Comment by deep-soft** [comment]
https://github.com/rustdesk/rustdesk/pull/4115
